tests/e2e-inst-test.sh

```diff
@@ -91,8 +91,8 @@ for TEST in $TESTS; do
         --output option:parse-arguments \
         --log file:$SCRIPT_TMP_DIR/tracee-log-$$ \
         --signatures-dir $SIG_DIR \
-        --filter comm=echo,mv,ls,tracee \
-        --filter set=signatures &
+        --scope comm=echo,mv,ls,tracee \
+        --event signatures &
```

Comment:
I wonder if we better call the flag `events` and not `event`. WDYT?

Reply:
Since `--scope` only defines one type of scope per flag, its name is fine. `--event` in turn can define many events or sets per flag, so perhaps `--events` better explains its intent.

Reply:
I didn't changed this yet. Waiting for feedback.

Reply:
We've decided offline to make it plural.